ceno_recursion/Cargo.toml

```diff
@@ -33,37 +33,26 @@ tracing-subscriber.workspace = true
-openvm-circuit = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-continuations = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-instructions = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-native-circuit = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-native-compiler = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-native-compiler-derive = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-native-recursion = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-rv32im-circuit = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-openvm-sdk = { git = "https://github.com/scroll-tech/openvm.git", branch = "feat/v1.4.1-scroll-ext", default-features = false }
-
-# openvm = { path = "../../openvm-scroll-tech/crates/toolchain/openvm", default-features = false }
-# openvm-circuit = { path = "../../openvm-scroll-tech/crates/vm", default-features = false }
-# openvm-continuations = { path = "../../openvm-scroll-tech/crates/continuations", default-features = false }
-# openvm-instructions = { path = "../../openvm-scroll-tech/crates/toolchain/instructions", default-features = false }
-# openvm-native-circuit = { path = "../../openvm-scroll-tech/extensions/native/circuit", default-features = false }
-# openvm-native-compiler = { path = "../../openvm-scroll-tech/extensions/native/compiler", default-features = false }
-# openvm-native-compiler-derive = { path = "../../openvm-scroll-tech/extensions/native/compiler/derive", default-features = false }
-# openvm-native-recursion = { path = "../../openvm-scroll-tech/extensions/native/recursion", default-features = false }
-# openvm-rv32im-circuit = { path = "../../openvm-scroll-tech/extensions/rv32im/circuit", default-features = false }
-# openvm-sdk = { path = "../../openvm-scroll-tech/crates/sdk", default-features = false }
-
-openvm-stark-backend = { git = "https://github.com/openvm-org/stark-backend.git", tag = "v1.2.1", default-features = false }
-openvm-stark-sdk = { git = "https://github.com/openvm-org/stark-backend.git", tag = "v1.2.1", default-features = false }
 p3-monty-31 = { git = "https://github.com/Plonky3/Plonky3.git", rev = "539bbc84085efb609f4f62cb03cf49588388abdb" }
 
 [features]
 # bench-metrics = ["openvm-circuit/bench-metrics"]
 default = ["parallel", "nightly-features"]
+cuda = ["openvm-circuit/cuda", "openvm-native-circuit/cuda"]
```

Comment:
2 option for feature name
option 1: `cuda = ["openvm-circuit/cuda", "openvm-native-circuit/cuda"]`
option 2: `gpu = ["openvm-circuit/cuda", "openvm-native-circuit/cuda"]`

option 2 better as it aligned with existing base layer prover `gpu` flag.

We can aligned either `gpu` or `cuda` just better to be consistent